pasta_eln/GUI/form.py

```diff
@@ -227,7 +227,7 @@ def __init__(self, comm:Communicate, doc:dict[str,Any]):
     if self.flagNewDoc: #new dataset
       TextButton('Save && Next', self, [Command.FORM_SAVE_NEXT], buttonLineL, 'Save this and handle next')
     # autosave
-    if (Path.home()/'.pastaELN.temp').exists():
```

Comment:
Will exists() check on .pastaELN.temp throw error if files does not exists? Why to change to is_file and is_dir in general?

Reply:
- Exists() does not throw an error, it checks if a file exists. If it does, ...
-  Exists tests for files and directories, but in some cases you want that it is a directory, file. For example a image viewer wants that it is a file that ends with jpg.